deployments/dashboard/prometheus/prometheus.yml

```diff
@@ -6,4 +6,4 @@ scrape_configs:
     scrape_interval: 5s
 
     static_configs:
-      - targets: ['127.0.0.1:9616']
+      - targets: ['relay-eth2sub:9616']
```

Comment:
Can we include both `127.0.0.1` and `relay-eth2sub` here? This will allow re-using the config outside of docker-compose as well.